src/frontend/ui/ui_manager.rs

```diff
@@ -180,27 +198,43 @@ impl UiManager {
     fn save_to_file(&self) {
         let img = self.get_crop_image();
 
+        let app_model_sender = self.app_model_sender.clone();
         FileChooser::open(move |file| {
             if let Some(path) = file {
-                img.save(path).expect("Couldn't save image.");
+                match img.save(&path) {
+                    Ok(()) => {
+                        app_model_sender.spawn_oneshot_command(move || Command::Notify {
```

Comment:
utility function

Reply:
May I ask how you imagine a utility function here?

```rust
fn notify(app_model_sender: ComponentSender<AppModel>, msg: String, urgency: Urgency) {
    app_model_sender.spawn_oneshot_command(move || Command::Notify { msg, urgency });
}
```
that doesn't look that much better ;-;

Reply:
```rust
fn notify(&self, msg: Notification) {
    self.app_model_sender.spawn_oneshot_command(move || Command::Notify { ..msg });
}
```

Reply:
but maybe you want to use another urgency. Also I don't really see a benefit of a utility function here except less typing

Reply:
and we've got another problem:
Here, we're in a clojure, so we'd have to use this function signature:
```rust
fn notify(app_model_sender: ComponentSender<AppModel>, msg: String, urgency: Urgency)
```

Reply:
> but maybe you want to use another urgency.

Thats why msg also holds the urgency...

> Also I don't really see a benefit of a utility function here except less typing

That's the benefit of a utility function in general...

>  Here, we're in a clojure, so we'd have to use this function signature:

Why don't you add a struct combining msg and urgency?